app/views/users/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>User</h1>
+<%= render partial: "username", locals: { user: @user } %>
+<div>
+<h3>Bio</h3>
+<p><%=@user.bio%></p>
+</div>
+<ul class='user-posts'>
+<% @user.posts.each_with_index do |post, index| %>
+  <li>
+  <h4>Post #<%=index + 1%></h4>
+  <p><%=post.text%></p>
+  <span>Comments: <%= post.comments_counter%>, Likes: <%= post.likes_counter%></span>
+  </li>
+<% end %>
+</ul>
```

Comment:
- [ ] Please make sure to add a button at the bottom in the users/show page `See all Post` clicking it opens the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).